articles/hackeRnews-article-01.Rmd

```diff
@@ -0,0 +1,166 @@
+- 'Jobs' section where users can browse job offers
+
+In this document basic usage of the `hackeRnews` package will be presented along with some example use cases.
+
+## Endpoint wrappers
+
+The `hackeRnews` package provides user friendly wrappers around the [Official Hacker News API](https://github.com/HackerNews/API). Specific endpoints along with corresponding functions and their descriptions are presented in the table below:
+
+| Endpoint            	| `hackeRnews` function                       	| Description                                                    	|
+|---------------------	|---------------------------------------------	|----------------------------------------------------------------	|
+| /v0/item/<id>       	| `get_item_by_id(id=<id>)`                   	| Retrieves the item corresponding to specified id               	|
+| /v0/user/<username> 	| `get_user_by_username(username=<username>)` 	| Retrieves the user corresponding to the specified username     	|
+| /v0/maxitem         	| `get_max_item_id()`                         	| Retrieves to the id of the latest published item               	|
+| /v0/topstories      	| `get_top_stories_ids()`                     	| Retrieves the ids of the current top stories                   	|
+| /v0/newstories      	| `get_new_stories_ids()`                     	| Retrieves the ids of current new stories                       	|
+| /v0/beststories     	| `get_best_stories_ids()`                    	| Retrieves the ids of best new stories                          	|
+| /v0/askstories      	| `get_latest_ask_stories_ids()`              	| Retrieves the ids of latest *ask* stories                      	|
+| /v0/showstories     	| `get_latest_show_stories_ids()`             	| Retrieves the ids of latest *show* stories                     	|
+| /v0/jobstories      	| `get_latest_job_stories_ids()`              	| Retrieves the ids of latest *job* stories                      	|
+| /v0/updates         	| `get_updates()`                             	| Retrieves the ids of the items and usernames that were updated 	|
+
+The package does not only contain wrappers for the API endpoints but it also provides some additional functions that simplify some operations. For example instead of retrieving the ids of top stories with the `get_top_stories_ids()` and then looping over the vector retrieving each item one by one using the `get_item_by_id(id=<id>)` function, the user can just use the `get_top_stories()` function. Moreover, `hackeRnews` makes use of the [`future.apply`](https://github.com/HenrikBengtsson/future.apply) package to speed up collecting multiple items by fetching them in parallel as collecting 500 stories sequentially might take a long time.
+
+## Use cases
+Before proceeding to the use cases, the installation and setup of the package will be described. `hackeRnews` is available on CRAN and can be installed using the following function call:
```

Comment:
How about an installation section instead of that introduction?

Reply:
Agree, done.